demo/list-box-styling-demos.html

```diff
@@ -0,0 +1,66 @@
+<dom-module id="list-box-styling-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Customizing the appearance</h3>
+    <vaadin-demo-snippet id="list-box-styling-demos-custom-border">
+      <template preserve-content>
+        <dom-module id="custom-list-box" theme-for="vaadin-list-box">
+          <template>
+            <style>
+              :host([theme~="custom-border"]) {
+                border: 1px solid black;
+                height: 100px;
+              }
+            </style>
+          </template>
+        </dom-module>
+        <vaadin-list-box theme="custom-border">
+          <vaadin-item>Option one</vaadin-item>
+          <vaadin-item>Option two</vaadin-item>
+          <vaadin-item>Option three</vaadin-item>
+          <vaadin-item>Option four</vaadin-item>
+          <vaadin-item>Option five</vaadin-item>
+          <vaadin-item>Option six</vaadin-item>
+          <vaadin-item>Option seven</vaadin-item>
+          <vaadin-item>Option eight</vaadin-item>
+          <vaadin-item>Option nine</vaadin-item>
+          <vaadin-item>Option ten</vaadin-item>
+        </vaadin-list-box>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Customizing selected icon</h3>
+    <vaadin-demo-snippet id="list-box-styling-demos-custom-icon">
+      <template preserve-content>
+        <dom-module id="custom-listbox-icon" theme-for="vaadin-item">
+          <template>
+            <style>
+              :host([theme~="custom"][tabindex])::before {
```

Comment:
`[tabindex]` here is confusing. It should be removed as this still works the same without it.

Reply:
Done.